pkg/kanctl/actionset.go

```diff
@@ -755,7 +759,7 @@ func generateActionSetName(p *PerformParams) (string, error) {
 	return "", errMissingFieldActionName
 }
 
-func verifyRepositoryServerParams(repoServer *crv1alpha1.ObjectReference, crCli versioned.Interface, ctx context.Context) error {
+func verifyRepositoryServerParams(waitForRepoServerReady bool, repoServer *crv1alpha1.ObjectReference, crCli versioned.Interface, ctx context.Context) error {
```

Comment:
Not in this PR but we should change the order of arguments here. The convention is
```
context, clis..., other args...
```

Reply:
Done !